Magento_Theme/web/js/custom-script.js

```diff
@@ -0,0 +1,18 @@
+let noRouteParagraph = document.querySelector('p');
+
```

Comment:
pro tip:
try to use `const` instead of `let` every time. and change it to `let` only when you have to reassign a value